### DISCUSSION TITLE
Grafana not showing data for sensor or search node

### DISCUSSION THREAD

**Discussion by rsharris3** [Security-Onion-Solutions/securityonion#3465] [Unsupported Versions] [answered]
still learning security onion and setup a distributed architecture with manager, search and sensor node.  when viewing dashboards in Grafana, there isn't any data for the sensor or search node.  

Setup using the iso 2.3.30 on CentOS 7.  When running salt-call state.highstate, no failures and when so-status, all services are ok on all three nodes.

Any suggestions on how to troubleshoot?

Thanks in advance for any suggestions!

**Comment by palichx** [comment, marked as answer]
Hi 
check date and time on servers. :)

**Comment by rsharris3** [comment]
Time sync issues was it.  Thanks for the poke in the right direction!
